src/modules/m_banredirect.cpp

```diff
@@ -259,7 +267,7 @@ class ModuleBanRedirect : public Module
 
 				for(BanRedirectList::iterator i = redirects->begin(); i != redirects->end(); i++)
 				{
-					modestack.Push('b', i->targetchan.insert(0, i->banmask));
+					modestack.Push('b', i->banmask + i->targetchan);
```

Comment:
Refrain from making superfluous changes in 2.0

Reply:
This change is necessary as the `iterator` in `std::set` is `const_iterator` so `i->targetchan` can not be modified, and `std::string::insert` modifies the string.